### DISCUSSION TITLE
Welcome to WWIDesigner Discussions!

### DISCUSSION THREAD

**Discussion by edwardkort** [edwardkort/WWIDesigner#104] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Please use this interface, rather than Issues, for asking questions: issues are hidden when resolved.

**Comment by JeffBurris** [comment]
FanTAStic - Thanks for WID overall and thanks for opening up this Discussion area. I have benefitted tremendously elsewhere and made invaluable acquaintances, but without sub-categories for this type of thing a WID question can be a bumpy ride. This will be perfectly suited for the task, thanks!

**Reply by edwardkort** [reply]
Thanks, Jeff. And as there are more discussion posts, I will create categories so that topics are easier to find.
